epub33/core/index.html

```diff
@@ -7492,13 +7492,104 @@ <h4>Content document dimensions</h4>
+												U+007F).</caption>
+
+											<tr>
+												<td id="viewport.ebnf.def">
+													<a href="viewport.ebnf.def">viewport</a>
+												</td>
+												<td>=</td>
+												<td>
+													( <a href="#viewport.ebnf.width">width</a>, 
+													  <a href="#viewport.ebnf.sep">sep</a>,
+													  <a href="#viewport.ebnf.height">height</a>
+													)
+													|
+													( <a href="#viewport.ebnf.height">height</a>, 
+													  <a href="#viewport.ebnf.sep">sep</a>,
+													  <a href="#viewport.ebnf.width">width</a>
+													)
+													;
+												</td>
+											</tr>
+											<tr>
+												<td id="viewport.ebnf.width">
+													<a href="#viewport.ebnf.width">width</a>
+												</td>
+												<td>=</td>
+												<td>"width", 
+													<a href="#viewport.ebnf.assign">assign</a>, 
+													( <a href="#viewport.ebnf.dimension">dimension</a> | "device-width" ) ;</td>
+											</tr>
+											<tr>
+												<td id="viewport.ebnf.height">
+													<a href="#viewport.ebnf.height">height</a href="#viewport.ebnf.height">
+												</td>
+												<td>=</td>
+												<td>"height" , 
+													<a href="#viewport.ebnf.assign">assign</a>, 
+													( <a href="#viewport.ebnf.dimension">dimension</a> | "device-height" ) ;</td>
+											</tr>
+											<tr>
+												<td id="viewport.ebnf.sep">
+													<a href="#viewport.ebnf.sep">sep</a>
+												</td>
+												<td>=</td>
+												<td>
+													[space], (";" | "," | space), [space]
+												</td>
+											</tr>
+											<tr>
+												<td id="viewport.ebnf.assign">
+													<a href="#viewport.ebnf.assign">assign</a>
+												</td>
+												<td>=</td>
+												<td>[space], "=", [space] ;</td>
+											</tr>
+											<tr>
+												<td id="viewport.ebnf.dimension">
+													<a href="#viewport.ebnf.dimension">dimension</a>
+												</td>
+												<td>=</td>
+												<td>? positive number ? ;</td>
+											</tr>
+											<tr>
+												<td id="viewport.ebnf.space">
+													<a href="#viewport.ebnf.space">space</a>
+												</td>
+												<td>=</td>
+												<td>#x20 ;</td>
+											</tr>
+										</table>
+									</dd>
+								</dl>
+
+								<p>The <code>device-width</code> and <code>device-height</code> values refer to the 100% of the width, respectively height, of the reading system's [=viewport=].</p>
+
+								<p class="note">
+									Earlier iterations of EPUB referred to the 
+									<a href="https://www.w3.org/TR/css-device-adapt-1/#viewport-meta"><code>meta</code> element syntax</a> [[css-device-adapt-1]] for the
+									the definition of the attribute value, which includes features that are not relevant for EPUB use. All aspects of that syntax that
+									is not defined by this section are <a href="#deprecated">deprecated</a>.
```

Comment:
What does it mean to deprecate in a note?

If it's not enforced, then why doesn't the ebnf allow the properties? And if it is enforced, then it's only a warning if you use the properties but, again, the ebnf makes anything else in the content attribute invalid.